DigitalRain.sln

```diff
@@ -3,18 +3,24 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.32510.428
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DigitalRain", "DigitalRain.csproj", "{6737A910-DA74-48D4-AAEF-CB53E976D947}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DigitalRain", "DigitalRain\DigitalRain.csproj", "{13A5D56B-9E65-4D2C-BFF3-97907E518694}"
```

Comment:
I like this change. I think when we setup the project you checked the box that said "place solution in project root" or something. But now we can organize our project a little better.

Reply:
Yeah.  Now we have one solution but two projects.  The Test project has a reference to the actual code project so that it can import and test it.